services/question-service/entry-points/api/routes.ts

```diff
@@ -0,0 +1,63 @@
+import express from 'express';
+import * as questionUseCase from '../../domain/question-use-case';
+import { validateAddQuestionInput, validateUpdateQuestionInput } from './validators';
+
+export default function defineRoutes(expressApp: express.Application) {
+  const router = express.Router();
+
+  router.put('/',validateAddQuestionInput ,async (req, res, next) => {
```

Comment:
This should be a post request, since you're creating a question rather than editing
```suggestion
  router.post('/', validateAddQuestionInput, async (req, res, next) => {
```